conanfile.py

```diff
@@ -188,8 +199,14 @@ def configure(self):
         if self.settings.os != "Linux":
             self.options.with_libalsa = False
 
+        if (not self.settings.shared) and self.settings.qtwebengine:
```

Comment:
did you mean `self.options["qt"]` instead of `self.settings` ?

Reply:
I probably did